packages.dhall

```diff
@@ -318,6 +318,31 @@ let additions =
         , repo = "https://github.com/mlabs-haskell/purescript-plutus-types"
         , version = "v1.0.1"
         }
+      , uplc-apply-args =
+        { dependencies =
+          [ "aff"
+          , "bytearrays"
+          , "cardano-serialization-lib"
+          , "cardano-types"
+          , "effect"
+          , "either"
+          , "foldable-traversable"
+          , "foreign-object"
+          , "js-bigints"
+          , "lists"
+          , "maybe"
+          , "mote"
+          , "mote-testplan"
+          , "partial"
+          , "prelude"
+          , "profunctor"
+          , "spec"
+          , "transformers"
+          , "tuples"
+          ]
+        , repo = "https://github.com/mlabs-haskell/purescript-uplc-apply-args"
+        , version = "master"
```

Comment:
We need to pin the version. Make an 'official' release by adding and pushing a tag.

Reply:
Addressed in [b64d0dd](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1627/commits/b64d0ddbc825ae449dab612222b4c2b574eee105)